src/combinatorics/simple_graph/connectivity.lean

```diff
@@ -394,7 +397,7 @@ instance connected_components.inhabited [inhabited V] : inhabited G.connected_co
 
 lemma connected_component.subsingleton_of_connected (h : G.connected) :
   subsingleton G.connected_component :=
-⟨λ c d, quot.ind (λ v d, quot.ind (λ w, quot.sound (h v w)) d) c d⟩
+⟨λ c d, quot.ind (λ v d, quot.ind (λ w, quot.sound (h.1 v w)) d) c d⟩
```

Comment:
This lemma should take `G.preconnected`

There should be another lemma that `nonempty G.connected_component` for `G.connected` too.  There should also probably be something that `unique G.connected_component` given `G.connected`, but I'm not sure the best way to organize this.